caikit/runtime/http_server/http_server.py

```diff
@@ -558,9 +628,9 @@ def _train_add_unary_input_unary_output_handler(self, rpc: ModuleClassTrainRPC):
         pydantic_response = dataobject_to_pydantic(response_data_object)
 
         @self.app.post(
-            get_http_route_name(rpc.name),
+            get_http_route_name(rpc_name=rpc.name),
             responses=self._get_response_openapi(
-                response_data_object, pydantic_response
+                response_data_object, pydantic_response, 202
```

Comment:
I think we've discussed usage of 202 vs 200 in the past. You're right that 202 is the right code for creating a job that is accepted by the server. On the flip side, I worry that using 202 could break `if response.code != 200` style checks on the client side (which are NOT correct, but almost certainly exist in places).

Reply:
Furthermore, I think this would be an API breaking change, so I think we should not move to 202 at this point.